src/commands/new.ts

```diff
@@ -104,7 +103,6 @@ export default {
       // generate the project
       startSpinner("Igniting app")
       await spawnProgress(log(expoCLIString), {
-        env: cliEnv,
```

Comment:
I removed this argument because [it is not currently used in spawnProgress](https://github.com/infinitered/ignite/blob/master/src/tools/spawn.ts#L5-L19)